### ISSUE TITLE
error loading mail widget

### ISSUE THREAD

**Issue by mckaygerhard** [fjies/dashboard#117]
i cant load and use the mail widget, no log action are performed in log of oc, app does not show me related info for configure, i just push the icon from apps to enabled and when goes to load dissapear..

**Comment by mckaygerhard** [comment]
i enable debug level and get some info :

´´´
{"reqId":"5500bd1826e09","app":"dashboard","message":"host oder user are missing for mail widget","level":2,"time":"2015-03-11T22:09:28+00:00","method":"POST","url":"\/simplenet\/index.php\/apps\/dashboard\/widget\/complete"}
{"reqId":"5500bd1826e09","app":"dashboard","message":"could not connect to mailbox in mail widget","level":3,"time":"2015-03-11T22:09:28+00:00","method":"POST","url":"\/simplenet\/index.php\/apps\/dashboard\/widget\/complete"}
{"reqId":"5500bd1826e09","app":"PHP","message":"Call to undefined function OCA\\Dashboard\\Widgets\\Mail\\imap_last_error() at \/var\/www\/simplenet\/apps\/dashboard\/widgets\/mail\/mailcontroller.php#121","level":3,"time":"2015-03-11T22:09:28+00:00","method":"POST","url":"\/simplenet\/index.php\/apps\/dashboard\/widget\/complete"}

´´´ 

the mail configuration are performed from oc config and users mail ?

**Comment by fjies** [comment]
no, you have to tell the widget in the widget settings what data to connect

**Comment by mckaygerhard** [comment]
the setting are same as the others widget i ssupost.. 

if them, so happened that when i added the mail widget the widget does not appear in the page. so i cannot configure the widget..

i revise the code .. searching for any php 5.4 related... i'm using php 5.3 ...
